### ISSUE TITLE
using too many unused variables inside code base why?

### ISSUE THREAD

**Issue by pooooriya** [MiaadTeam/lesan#172]
![image](https://github.com/user-attachments/assets/6446ec99-07a7-457c-bf36-cddb44d6c763)
![image](https://github.com/user-attachments/assets/9ac42401-f46d-4185-a278-5bed443540d8)

**Comment by hemedani** [comment]
Yeah, I should be creating a new branch for the new concept, But these variables exist to create transactions. These will be used when transactions are implemented.
Thank you for your notice, I will try to implement the transactions soon.
